extensions/amp-story/1.0/page-advancement.js

```diff
@@ -610,7 +610,7 @@ export class ManualAdvancement extends AdvancementConfig {
    */
   maybePerformNavigation_(event) {
     const target = dev().assertElement(event.target);
-    const pageRect = this.element_.getLayoutBox();
+    const pageRect = this.element_.querySelector('amp-story-page[active]').getLayoutBox();
```

Comment:
I could be wrong, but isn't `this.element_` always the active page here?

Reply:
Haha you wrote this change 👀

Reply:
Shhh... no one saw this 🤫